Dockerfile

```diff
@@ -17,11 +17,11 @@ RUN apt-get update \
  && pip install -r build/requirements.txt \
  && python setup.py install 
 
-FROM quay.io/ncigdc/apache-base:2.4.18-1.0.0
+FROM quay.io/ncigdc/apache-base:1.0.3
 
 LABEL org.label-schema.name="indexd" \
       org.label-schema.description="indexd container image" \
-      org.label-schema.version="1.0.0" \
+      org.label-schema.version="1.0.1" \
```

Comment:
Looks like our versioning for this project got out of sync. :confused: Can we do 2.2.0 here and bump it up in`setup.py` as well?

Reply:
We could also merge this and then adjust the version in a separate PR.